client/state/notices/account-recovery/index.js

```diff
@@ -39,7 +39,10 @@ const getUpdateSuccessMessage = ( target, getState ) => {
 			return translate( 'Successfully updated recovery email address.' );
 
 		default:
-			return translate( 'Successfully updated the recovery option.' );
+			return translate( 'Successfully updated the recovery option.', {
+				comment: 'Recovery option here is a generic term for all the ways for account recovery.' +
```

Comment:
Oh! I didn't know there is `make translate`. TIL :)
I've stolen your string in a9bac7d447a0716252762d7d20bca26019f499f2, and they did show up in `calypso-strings.bot`